executor/executor.go

```diff
@@ -363,16 +363,6 @@ func (e *CheckTableExec) Open(ctx context.Context) error {
 	return nil
 }
 
-// Next implements the Executor Next interface.
-func (e *CheckTableExec) Next(ctx context.Context) (Row, error) {
-	if e.done {
-		return nil, nil
-	}
-	err := e.run(ctx)
```

Comment:
we can remove `e.run` and put its code inside `NextChunk`

Reply:
ok